### ISSUE TITLE
Spotify incorrect album art url

### ISSUE THREAD

**Issue by HolyStephano** [altdesktop/playerctl#172]
The player I am using is Spotify.

Reproduce:

run `playerctl metadata`
look at the album art url and open it, see incorrect URL.

Expected Results: 

Get the correct URL.

Not sure if this is just another Spotify issue (wouldn't surprise me) but the URL doesn't actually link to an image anymore, the resource isn't found.

**Comment by SwiftyChicken** [comment]
It seems like its a Spotify issue. They have probably moved or removed all the images from  https://open.spotify.com/image/{art-address}.

But I have found a fix for it without needing to use the Spotify API. Thanks to a Github project I found:
https://github.com/busybox11/NowPlaying-for-Spotify

It seems that it uses this site: 
https://i.scdn.co/image/{art-address}
Instead using the one playerctl use.
I don't now if that site is owned by Spotify or someone else but at the moment it works for me and my scripts I need it for.

So if you want to get the cover image use simply run: 
`playerctl metadata | sed "s/open\.spotify\.com/i.scdn.co/"`

So hopefully this might help you or others with the same problem.
